packs/vsphere/actions/vm_hw_basic_build.yaml

```diff
@@ -0,0 +1,60 @@
+---
+name: "vm_hw_basic_build"
+runner_type: "mistral-v2"
+pack: "vsphere"
+enabled: true
+entry_point: 'workflows/vm_hw_basic_build.yaml'
+description: WorkFlow to build a base VM hardware and power it on (CPU, RAM, HDD, NIC)
+
+parameters:
+  vm_name: 
+    required: true
+    type: string 
+  vm_hdd_dscluster: 
+    type: string
+    description: Which Datastore Cluster should be used for this HDD. Blank = same as Base VM    
+  vm_resourcepool: 
+    type: string
+  vm_version: 
+    default: vmx-09
```

Comment:
I left V10 in the action because it was the latest. in the basic build i put it as 9 as i'm using the desktop client for vsphere and it doesn't allow edit's on v10 vm's through the desktop app.